bin/apply.bash

```diff
@@ -19,14 +19,16 @@ source venv/bin/activate
 pip install -r requirements.txt
 
 log_info "Running kubespray"
-sops_exec_file_no_fifo "${secrets[ssh_key]}" "ansible-playbook -i "${config[inventory_file]}" cluster.yml -b --private-key {}"
+sops_exec_file_no_fifo "${secrets[ssh_key]}" "ansible-playbook -i "${config[inventory_file]}" cluster.yml -b --private-key {} ${@}"
```

Comment:
Would it be possible for this script to only run `./run-playbook.bash cluster.yml ${@}`?
Not part of this issue, but just a thought that popped up.

Reply:
Hmm, yeah that should work. The only difference should be some of the terminal messages

Reply:
I think it would be nice. But if you don't want to do it here we can always add another PR for this

Reply:
Sounds good. I'll merge this now then.